HousingSearchApi/V1/Factories/QueryablePerson.cs

```diff
@@ -0,0 +1,113 @@
+using System.Collections.Generic;
+using HousingSearchApi.V1.Domain;
+using Nest;
+using Identification = HousingSearchApi.V1.Domain.ElasticSearch.Identification;
+using Tenures = HousingSearchApi.V1.Domain.ElasticSearch.Tenures;
+
+namespace HousingSearchApi.V1.Factories
+{
+    public class QueryablePerson
+    {
+        public Person Create()
+        {
+            return new Person
+            {
+                Id = Id,
+                Title = Title,
+                Firstname = Firstname,
+                MiddleName = MiddleName,
+                Surname = Surname,
+                PreferredFirstname = PreferredFirstname,
+                PreferredSurname = PreferredSurname,
+                Ethinicity = Ethinicity,
+                Nationality = Nationality,
+                PlaceOfBirth = PlaceOfBirth,
+                DateOfBirth = DateOfBirth,
+                Gender = Gender,
+                Identification = Create(Identification ?? new List<Identification>()),
+                PersonTypes = PersonTypes,
+                IsPersonCautionaryAlert = IsPersonCautionaryAlert,
+                IsTenureCautionaryAlert = IsTenureCautionaryAlert,
+                Tenures = Create(Tenures ?? new List<Tenures>())
+            };
+        }
+
+        private static List<Domain.Identification> Create(List<Identification> identifications)
```

Comment:
Should this be part of `QueryablePerson` class?

Reply:
I think it's convenient enough to do so, since Identification/Tenures have no meaning on their own as entities. Here they can just be used to create the DomainObject they belong to.